src/log.c

```diff
@@ -149,6 +153,8 @@ static log_sink_t* log_sink_lh(const char* name) {
 }
 
 
+typedef enum {DEFAULT, NONE, TRACE, DEBUG, INFO, NOTICE, WARNING, ERROR, CRITICAL, N_LEVELS} level_index_t;
```

Comment:
Why are there histograms for DEFAULT and NONE?  These will never be raised in an actual log.